types/ensure/service.pp

```diff
@@ -0,0 +1,7 @@
+# https://docs.puppet.com/puppet/latest/types/service.html#service-attribute-ensure
+# (Property: This attribute represents concrete state on the target system.)
+#
+# Whether a service should be running.
+#
+# Valid values are stopped (also called false), running (also called true).
+type Tea::Ensure::Service = Variant[Enum['stopped','running'],Boolean]
```

Comment:
https://github.com/puppetlabs/puppetlabs-stdlib/blob/master/types/ensure/service.pp is a bit more opinionated but I'd prefer to standardize on that type instead of adding it here.

Reply:
Agreed :+1: